### PR TITLE
Change redirection to display only messages for specified cinema

### PR DESCRIPTION
**Pull request by jiangyue12392**
Solving issue: #40 
@Reine0017 I removed the `href` link on the button to do the redirection in javascript since I need to pass on the message of which cinema location to the `feed.html` page. The button seems to be not styled again possibly because it is no longer a `href` element. Please look into it and restore the styling of the `submit review` button

### CODE REVIEW

File: src/main/webapp/js/messages-loader.js [hunk 1]
```diff
@@ -48,38 +50,21 @@ function buildMessageDiv(message){
...
-        messageContainer.innerHTML = '<p>There are no posts yet.</p>';
-      } else {
-        messageContainer.innerHTML = '';
-      }
-      messages.forEach((message) => {
-        messageContainer.appendChild(buildMessageDiv(message));
-      });
-    } else {
-      messageContainer.innerText = 'Translation failed. Please refresh the page.';
-      console.log(transReq.status);
```
Comment (Reine0017) [inline src/main/webapp/js/messages-loader.js:77]: Do you want to leave this here?

**Comment by Reine0017** [comment]
> The button seems to be not styled again possibly because it is no longer a `href` element. Please look into it and restore the styling of the `submit review` button

Okay, thanks alot for letting me know this part!

File: src/main/webapp/js/messages-loader.js (hunk 1 above)
Comment (jiangyue12392) [reply src/main/webapp/js/messages-loader.js:77]: That’s a removal

Comment (jiangyue12392) [reply src/main/webapp/js/messages-loader.js]: That’s a removal

Comment (Reine0017) [reply src/main/webapp/js/messages-loader.js:77]: oh *facepalm* my bad i missed it

**Review by Reine0017 — APPROVED** [review]
Looks good!

**Pull request merged**